### DISCUSSION TITLE
Jumping to diagnostics doesn't work

### DISCUSSION THREAD

**Discussion by mfilej** [nvim-neotest/neotest#239] [Q&A]
Hello! I've just tried neotest, and I'm really excited about how well it works. One thing I haven't managed to get working yet is how to see the output of a diagnostic message. Out of the box, I'm able to see the symbols in the sidebar like I do for any other kind of diagnostics — however, I can't jump to them like I usually do with `lua vim.diagnostic.goto_next()` for example — which would then show a popup message.

I couldn't understand if this requires additional config. I've seen mentions of       `vim.api.nvim_create_namespace("neotest")`, but it didn't make a difference (does that line go before `require('neotest').setup` or after?).

edit: I forgot to mention that the resulting message after using `goto_next()` is `No more valid diagnostics to move to`

**Comment by rcarriga** [comment]
I'm not sure why this wouldn't work, it's how I interact with them in my setup. I'd recommend using the bug report's minimal init.lua to try narrow down the issue
